application-tls-termination/manifests/traefik-ingress-ums.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: traefik.io/v1alpha1
+kind: ServersTransport
+metadata:
+  name: ums-backend-transport
+  namespace: howso
+spec:
+  rootCAsSecrets:
+    - platform-app-tls-ca 
+  serverName: platform-ums.default.svc.cluster.local
+  insecureSkipVerify: false 
+
+---
+apiVersion: traefik.io/v1alpha1
+kind: IngressRoute
+metadata:
+  name: platform-ums-management
+  namespace: howso
+spec:
+  entryPoints:
+    - websecure
+  routes:
+    - match: Host(`management.local.howso.com`)
+      kind: Rule
+      services:
+        - name: platform-ums
+          port: 8443
+          serversTransport: ums-backend-transport
+      middlewares:
+        - name: platform-ums-redirect
+  tls:
+    secretName: platform-ingress-tls
+
+---
+apiVersion: traefik.io/v1alpha1
+kind: IngressRoute
+metadata:
+  name: platform-ums-api
+  namespace: howso
+spec:
+  entryPoints:
+    - websecure
+  routes:
+    - match: Host(`api.local.howso.com`) && PathPrefix(`/api/v2/manage/`)
```

Comment:
is this supposed to be /api/v2/manage ?  or v3?  I thought we were moving to api/v3; perhaps not here.

Reply:
confirmed with @fulpm that v2 is correct